src/components/common/Alpha.tsx

```diff
@@ -13,16 +16,19 @@ interface Props {
 }
 
 export const Alpha = ({ className, hsva, onChange }: Props): JSX.Element => {
-  const handleMove = useCallback(
-    (interaction: Interaction) => {
-      // Alpha always fit into [0, 1] range
-      onChange({ a: interaction.left });
-    },
-    [onChange]
-  );
+  // Alpha always fit into [0, 1] range
+  const handleMove = (interaction: Interaction) => {
+    onChange({ a: interaction.left });
+  };
+
+  const handleKey = (offset: Interaction) => {
+    onChange({ a: clamp(hsva.a + offset.left) });
+  };
```

Comment:
Should the comment (`// Alpha always...`) be moved down to be above the usage of `clamp()`?